src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -317,36 +317,39 @@ const ProfilePageContent = ({
       {user && !user.isCoordinapeUser && (
         <div className={classes.sections}>
           <Section title="My Circles">
-            {profile?.users?.map(u =>
-              u.circle ? (
-                <div key={u.id} className={classes.circle}>
-                  <Avatar
-                    alt={u.circle.name}
-                    src={
-                      u.circle?.logo ? getAvatarPath(u.circle?.logo) : undefined
-                    }
-                  >
-                    {u.circle.name}
-                  </Avatar>
+            {profile?.users?.map(
+              u =>
+                u.circle && (
```

Comment:
I replaced a ternary with `&&` to make this simpler.

Reply:
Thank you